### PR TITLE
Trigger eager loading when using the attributes serializer option

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

TODO:

* [x] Add a test

### CODE REVIEW

File: src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php [hunk 1]
```diff
@@ -137,7 +137,17 @@ private function joinRelations(QueryBuilder $queryBuilder, QueryNameGeneratorInt
...
 
-            if ((false === $propertyMetadata->isReadableLink() || false === $propertyMetadata->isReadable()) && false === $propertyMetadata->getAttribute('fetchEager', false)) {
+            if ($inAttributes = isset($context[AbstractNormalizer::ATTRIBUTES][$association])) {
+                // prepare the child context
+                $context[AbstractNormalizer::ATTRIBUTES] = $context[AbstractNormalizer::ATTRIBUTES][$association];
+            } else {
+                unset($context[AbstractNormalizer::ATTRIBUTES]);
+            }
+
+            if (
```
Comment (Simperfit) [inline src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php]: cs is strange here

File: src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php [hunk 2]
```diff
@@ -137,7 +137,17 @@ private function joinRelations(QueryBuilder $queryBuilder, QueryNameGeneratorInt
...
+                ((!$inAttributes && false === $propertyMetadata->isReadableLink()) || false === $propertyMetadata->isReadable()) &&
+                false === $propertyMetadata->getAttribute('fetchEager', false)
```
Comment (soyuka) [inline src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php]: ` false === $propertyMetadata->getAttribute('fetchEager', false)` should not be needed here as it should've been tested https://github.com/dunglas/core/blob/master/src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php#L142 

Did you start you PR on master? I can't recognize the replaced code which may have lead to weird copy/paste?

File: src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php (hunk 1 above)
Comment (dunglas) [reply src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php]: Why?

File: src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php (hunk 2 above)
Comment (dunglas) [reply src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php]: Can I let you handle that? I've changed this part (only the CS) and I'm not really comfortable with it.

**Pull request merged**